<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegWindow</class>
 <widget class="QDialog" name="RegWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>254</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>254</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IDS_REG_WINDOW</string>
  </property>
  <widget class="QLabel" name="labelUserName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IDS_LABEL_USERNAME</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editUserName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>153</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>IDS_LABEL_PASSWORD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editPassword">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>153</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listUserIcon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>233</width>
     <height>148</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>false</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragDrop</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="viewMode">
    <enum>QListView::IconMode</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="editRePassword">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>153</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelRePassword">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>IDS_LABEL_REPASSWORD</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRePassword_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>IDS_LABEL_USERICON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOK">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>IDS_BUTTON_OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>IDS_BUTTON_CANCEL</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editUserName</tabstop>
  <tabstop>editPassword</tabstop>
  <tabstop>editRePassword</tabstop>
  <tabstop>listUserIcon</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
